### ISSUE TITLE
Unable to see complete errors / messages...

### ISSUE THREAD

**Issue by uc-msft** [Microsoft/azuredatastudio#2836]
Issue Type: <b>Bug</b>

Debugging TSQL scripts in ADS is hard since it doesn't show all the errors or messages from the server. This is extremely confusing and incorrect behavior.

**Repro:**
```
exec sp_execute_external_script 
@language = N'Python',
@script= N'-- just fail the script'
```

**Output (ADS):**
1:51:47 PMStarted executing query at Line 1
Msg 39004, Level 16, State 20, Line 0 A 'Python' script error occurred during execution of 'sp_execute_external_script' with HRESULT 0x80004004.
Msg 39019, Level 16, State 2, Line 0 An external script error occurred: Error in execution. Check the output for more information.
Total execution time: 00:00:03.258

**Actual Output from server (SSMS):**
Msg 39004, Level 16, State 20, Line 0
A 'Python' script error occurred during execution of 'sp_execute_external_script' with HRESULT 0x80004004.
Msg 39019, Level 16, State 2, Line 0
An external script error occurred: 

Error in execution.  Check the output for more information.
Traceback (most recent call last):
  File "<string>", line 3, in <module>
  File "/var/opt/mssqlext/data/b21332bf-3ce9-497d-bcc7-1911a520839f/AF01599F-1CBE-440F-83EF-E7AB8F6E7470/sqlindb_0.py", line 27
    -- just fail the script
               ^
SyntaxError: invalid syntax

SqlSatelliteCall error: Error in execution.  Check the output for more information.
STDOUT message(s) from external script: 
SqlSatelliteCall function failed. Please see the console output for more information.
Traceback (most recent call last):
  File "/opt/mssql/mlservices/libraries/PythonServer/revoscalepy/computecontext/RxInSqlServer.py", line 605, in rx_sql_satellite_call
    rx_native_call("SqlSatelliteCall", params)
  File "/opt/mssql/mlservices/libraries/PythonServer/revoscalepy/RxSerializable.py", line 375, in rx_native_call
    ret = px_call(functionname, params)
RuntimeError: revoscalepy function failed.

Azure Data Studio version: azuredatastudio 1.0.0 (cab8f3e37c9ac4367e0dacef5185da08bd8021c2, 2018-09-20T22:04:35.897Z)
OS version: Windows_NT x64 10.0.18252

**Comment by MattIrv** [comment]
This looks like a bug we fixed before but apparently is back now: https://github.com/Microsoft/azuredatastudio/issues/528

**Comment by MattIrv** [comment]
I'm having a bit of trouble reproducing this but I'll keep looking. When I run the command given to repro it I actually get the correct output but it is missing line breaks so the only way to view it is to copy and paste it. I'll fix that bug too

**Comment by MattIrv** [comment]
@uc-msft do you remember what version of SQL Server you were connected to? I can't reproduce this bug when connected to SQL Server 2017 but it may be a version-specific thing

**Comment by MattIrv** [comment]
I fixed the issue where stack traces and other messages had line breaks removed in https://github.com/Microsoft/azuredatastudio/pull/2935 but still haven't been able to repro the original issue

**Comment by MattIrv** [comment]
Closing this as fixed by #2935, if you're still seeing similar issues please reopen
